pkg/gitparse/gitparse_test.go

```diff
@@ -20,6 +29,20 @@ type testCase struct {
 	function func(bool, ParseState, []byte) bool
 }
 
+func TestParsePanicDiff(t *testing.T) {
+	// setup logger
+	logger, sync := log.New("forager/scanner",
+		log.WithConsoleSink(os.Stderr),
+	)
+	defer func() { _ = sync() }()
+	context.SetDefaultLogger(logger)
```

Comment:
This log setup can probably be removed - we now have a default global logger that will be used from `context.Background()`.